### PR TITLE
Issues 4 mytiaiev

### PR DESCRIPTION
**Pull request by Mytiaiev**
create clear clone

add handler for login. authorization. registration, today reports.

### CODE REVIEW

**Comment by SergeyKutsko** [comment]
@Mytiaiev any test cases?

**Comment by Mytiaiev** [comment]
add test_bot.py, consist basic test case

File: scripts/bot.py [hunk 1]
```diff
@@ -66,14 +72,102 @@ def error_handler(update: object, context: CallbackContext) -> None:
...
     updater = Updater(os.environ['TELEGRAM_TOKEN'], use_context=True)
     dp = updater.dispatcher
-    dp.add_handler(CommandHandler("start", start))
+    dp.add_handler(CommandHandler("start",  authorization))
+    dp.add_handler(CommandHandler("help", get_help))
     dp.add_handler(CommandHandler("report", report, run_async=True))
     dp.add_handler(CommandHandler("save_reports", save_reports))
-    dp.add_handler(MessageHandler(Filters.text, code))
+    dp.add_handler(MessageHandler(Filters.text('code'), code))
+    dp.add_handler(MessageHandler(Filters.text('Get registration'), reg_handler ))
```
Comment (SergeyKutsko) [inline scripts/bot.py:166]: @Mytiaiev what a purpose of empty handler? please remove

File: scripts/bot.py [hunk 2]
```diff
@@ -13,21 +13,27 @@
...
+from app.models import *
+from . import bolt, uklon, uber
 
 sys.path.append('app/libs')
 from app.libs.selenium_tools import get_report
 PORT = int(os.environ.get('PORT', '8443'))
 
 DEVELOPER_CHAT_ID = 803129892
 
+username = ''  # <- Global username take argument from foo authorization
```
Comment (SergeyKutsko) [inline scripts/bot.py:25]: @Mytiaiev now we have all models we need to replace with Models all this variables.

**Comment by SergeyKutsko** [comment]
@Mytiaiev replace variables with Models which are added before schedule Interview call by
https://calendly.com/universal_auto_internship/30min

File: scripts/bot.py [hunk 3]
```diff
@@ -13,21 +13,27 @@
...
+managers_list = []  # <- List of users to take all statistic day 
+uber_drivers_list = []# <- List of users who can request only self stat from Uber table
```
Comment (SergeyKutsko) [inline scripts/bot.py:27]: @Mytiaiev plese replace this variables by models.

Comment (Mytiaiev) [reply scripts/bot.py:27]: Sure, model User might be refactoring to ForigeinKey with model PaymantsOrder, and either next query might to PaymantsOrder ?

Comment (SergeyKutsko) [reply scripts/bot.py:27]: @Mytiaiev user should referencing Driver or Manager or Owner

Comment (Mytiaiev) [reply scripts/bot.py:27]: where from should be query with data?

Comment (SergeyKutsko) [reply scripts/bot.py:27]: @Mytiaiev from DB

**Comment by Mytiaiev** [comment]
replaced variables with Models which are added
All checks have passed
This branch has no conflicts with the base branch

**Review by SergeyKutsko — APPROVED** [review]
@Mytiaiev looks good, here is your Interview link: https://calendly.com/universal_auto_internship/30min

**Pull request merged**